config/betterquesting/DefaultQuests/Quests/BiofortheMasses-AAAAAAAAAAAAAAAAAAAAFQ==/TitaniumChemical-AAAAAAAAAAAAAAAAAAAMPA==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "This is the difficult part in the whole process. The multi in which Nitrobenzene is crafted is the Exxonmobil Chemical Plant, also known as the Chemplant, and it is a gigantic 7x7x7 structure with 4 separate upgradable parts to it. Even worse, the outer structure uses multiblock Machine Casings, making the DT look cheap by comparison, and the inside needs a lot of the casings used to craft single blocks.\n\nThe difference between those is that the tier of the outer structure that you need is mentioned in the recipe you want to craft, and the tier of the inner structure is generally related to the EU/t of the recipe, minus one tier. This is because, like the EBF and the LCR, you can use 2 energy hatches, so this HV recipe can be made with MV energy hatches. Since the inner structure\u0027s tier decides the max tier of the hatches used in the Chemplant, you can make it out of MV machine casings, which use Aluminium.\n\nThe outer structure will still need many stacks of Titanium, so this is where most of your effort will go. Good luck."
+      "desc:8": "This is the difficult part in the whole process. The multi in which Nitrobenzene is crafted is the Exxonmobil Chemical Plant, also known as the Chemplant, and it is a gigantic 7x7x7 structure with 4 separate upgradable parts to it. Even worse, the outer structure uses multiblock Machine Casings, making the DT look cheap by comparison, and the inside needs a lot of the casings used to craft single blocks.\n\nThe difference between those is that the tier of the outer structure that you need is mentioned in the recipe you want to craft, and the tier of the inner structure is generally related to the EU/t of the recipe, minus one tier. This is because, like the EBF and the LCR, you can use 2 energy hatches, so this HV recipe can be made with MV energy hatches. Since the inner structure\u0027s tier decides the max tier of the hatches used in the Chemplant, you can make it out of MV machine casings, which use Aluminium.\n\nThe outer structure will still need many stacks of Titanium, so this is where most of your effort will go. Good luck.\n\n§3Technically, you could also make Nitrobenzene in a LCR instead of a Chemplant. But it is less energy efficient and dilutes your acid. That option was only added for those to weak to handle a Chemplant."
```

Comment:
too* weak to handle a Chemplant.

Reply:
thanks! also _too_ late :)

Reply:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13867